examples/trials/mnist-nas/classic_mode/config_hopt.yml

```diff
@@ -0,0 +1,16 @@
+authorName: default
```

Comment:
why this file name is "config_hopt.yml "?

Reply:
ok, let me change the name to config_hpo.yml

Reply:
that is tuning NAS with HPO